zds/gallery/tests/tests_forms.py

```diff
@@ -67,7 +65,7 @@ def test_invalid_user_gallery_form_noexist_user(self):
 class ImageFormTest(TestCase):
 
     def test_valid_image_form(self):
-        upload_file = open(os.path.join(settings.BASE_DIR, 'fixtures', 'logo.png'), 'r')
+        upload_file = open(os.path.join(settings.BASE_DIR, 'fixtures', 'logo.png'), 'rb')
```

Comment:
Oui. Par contre avec tes derniers rebase tu as recassé les dépendances...

2016-03-24 11:51 GMT+01:00 Laville Augustin <notifications@github.com>:

> In zds/gallery/tests/tests_forms.py
> <https://github.com/zestedesavoir/zds-site/pull/3414#discussion_r57300512>
> :
>
> > @@ -67,7 +65,7 @@ def test_invalid_user_gallery_form_noexist_user(self):
> >  class ImageFormTest(TestCase):
> >
> >      def test_valid_image_form(self):
> > -        upload_file = open(os.path.join(settings.BASE_DIR, 'fixtures', 'logo.png'), 'r')
> > +        upload_file = open(os.path.join(settings.BASE_DIR, 'fixtures', 'logo.png'), 'rb')
>
> Pour le coup c'est nécessaire car Python a changé un poil à ce niveau si
> je ne dis pas de bêtises et on doit spécifier b lorsqu'on ouvre un
> fichier "binaire". À confirmer.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3414/files/7b8fb2573df042373c11b630b5456aa3d41ca7f3#r57300512>
>